src/main/archetype/ui.tests/test-module/lib/wdio-pixelmatch-service/launcher.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Do we need to add copyright headers here ?

Reply:
I am not sure whether open source project should have copyright header, @pankaj-parashar  can confirm

Reply:
@suryansh29 , please add copyright header to all js file added by us. please refer copyright header at https://github.com/adobe/aem-project-archetype/blob/develop/src/main/archetype/ui.tests/test-module/specs/aem/forms.js

Reply:
added